src/snowflake/snowpark/dataframe.py

```diff
@@ -520,6 +520,15 @@ def __init__(
     def stat(self) -> DataFrameStatFunctions:
         return self._stat
 
+    @property
+    def is_cached(self) -> bool:
+        """Whether the dataframe is cached."""
+        return self._is_cached
+
+    @is_cached.setter
+    def is_cached(self, value: bool):
+        self._is_cached = value
+
```

Comment:
Is this related to split view?

Reply:
Yes, this is a workaround for https://github.com/sphinx-doc/sphinx/issues/9884 of autosummary.

Reply:
Ok. The change looks good to me. Only concern is the breaking links which Mary mentioned above. Do you have a plan to deal with those?